internal/devbox/packages.go

```diff
@@ -456,6 +458,18 @@ func (d *Devbox) installNixPackagesToStore(ctx context.Context, mode installMode
 			if err == nil {
 				args.Env = creds.Env()
 			}
+
+			// We don't want to return an error here if configuring
+			// the cache fails. Just proceed without it.
+			u, err := user.Current()
+			if err != nil {
+				err = redact.Errorf("lookup current user: %v", err)
+				debug.Log("error configuring cache: %v", err)
+			}
+			err = d.providers.NixCache.Configure(ctx, u.Username)
+			if err != nil {
+				debug.Log("error configuring cache: %v", err)
+			}
```

Comment:
Should we use `ux.FWarning` and print top stderr?

debug doesn't get shown by default.